activestorage/test/dummy/config/puma.rb

```diff
@@ -28,7 +28,7 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" } if ENV.fetch("RAILS_ENV", "development") == "development"
```

Comment:
Same here. If someone specify the PIDFILE we should use it regardless of the environment.

```suggestion
if ENV["PIDFILE"]
  pidfile ENV["PIDFILE"]
else
  pidfile tmp/pids/server.pid" if ENV.fetch("RAILS_ENV", "development") == "development"
end
```